pkg/acs/calacs/acsrej/acsrej_init.c

```diff
@@ -196,25 +185,26 @@ int acsrej_init (IODescPtr ipsci[], IODescPtr ipdq[], clpar *par, int nimgs,
                         expn = exp2[ipts[dum / 2]];
                     }
                 }
-                raw0 = Pix(sg->sci.data, i, j);   /* DN/s */
+                raw0 = Pix(sg->sci.data, i, j);   /* e/s */
                 exp2n = (expn > 0.) ? expn : 1.;  /* s^2 */
                 if (newbias == 0) {
                     /* This is the 2nd term in DHB formula.
-                  https://trac.stsci.edu/ssb/stsci_python/ticket/1223#comment:10
-                       ERR = (READNSE^2 + VAL/GN? + (SCALE * VAL)^2) / EXPTIME^2
-                       nse is DN^2
-                       raw0 is DN/s - is /s supposed to be there?
+                  https://trac.stsci.edu/ssb/stsci_python/ticket/1223#comment:13
+                       ERR = (READNSE^2 + VAL + (SCALE * VAL)^2) / EXPTIME^2
+                       nse is e^2
+                       raw0 is e/s - is /s supposed to be there?
                        exp2n is s^2
                     */
                     Pix(sg->err.data, i, j) =
-                        (nse[0] + raw0 / gn[0] + SQ(scale * raw0)) / exp2n;
+                        (nse[0] + raw0 + SQ(scale * raw0)) / exp2n;
```

Comment:
From dimensional analysis alone, I do not see how the computation of Pix(sg->err.data,i,j) can be correct here, line 245, and line 314.  I will wait to see if this is clarified in later updates.

Reply:
Hmm, GitHub is saying that you commented on outdated code. Were you looking at an intermediate commit that has since been superseded by a subsequent one? That is, is this change still present in the final commit of the PR?